platform/linux-generic/odp_packet.c

```diff
@@ -1910,6 +1911,47 @@ int _odp_packet_copy_md_to_packet(odp_packet_t srcpkt, odp_packet_t dstpkt)
 	return dst_uarea_size < src_uarea_size;
 }
 
+static uint32_t packet_sum_ones_comp16_32(odp_packet_hdr_t *pkt_hdr,
+					  uint32_t offset,
+					  uint32_t len,
+					  odp_bool_t odd_offset)
+{
+	uint32_t sum = 0;
+
+	if (offset + len > pkt_hdr->frame_len)
+		return 0;
+
+	while (len > 0) {
+		uint32_t seglen = 0; /* GCC */
+		void *mapaddr = packet_map(pkt_hdr, offset, &seglen, NULL);
```

Comment:
check return code for packet_map() is missing here

Reply:
@muvarov This is an internal routine as is `packet_map()`. The only way `packet_map()` can fail is if `offset` is out of range, which we know isn't the case here. So I don't think the RC check is needed.

Reply:
ok